src/js/edu-benefits/5490/config/form.js

```diff
@@ -254,8 +256,8 @@ const formConfig = {
                 'transferOfEntitlement',
                 'veteranFullName',
                 'view:veteranId',
-                'other',
-                'view:noPreviousBenefits'
+                'view:noPreviousBenefits',
```

Comment:
Are you sure? Last I heard, this was supposed to go back to the top (last comment on department-of-veterans-affairs/vets.gov-team#1808)